lib/src/base-styles/_nx-btn.scss

```diff
@@ -7,111 +7,122 @@
 @import '../scss-shared/nx-variables';
 @import '../scss-shared/nx-text-helpers';
 
+%nx-btn--disabled {
+  &, &:hover, &:focus {
+    background-color: #e9e9e9;
+    border-color: #949494;
+    color: #949494;
+    cursor: default;
+  }
+}
+
 .nx-btn {
   @include semi-bold();
   @include container-horizontal;
 
-  background-color: $nx-grey-1;
-  border: $nx-border-dark;
-  border-radius: 4px;
-  color: $nx-text-color-light;
+  background-color: $nx-white;
+  border: 1px solid #045dd1;
+  border-radius: 6px;
+  color: #045dd1;
   cursor: pointer;
   display: inline-block;
-  font-size: 14px;
-  height: 32px;
+  font-size: 16px;
   line-height: 16px;
-  margin: 0 8px;
-  padding: 0 12px;
+  margin: 0 $nx-spacing-xs;
+  padding: $nx-spacing-sm $nx-spacing-l;
   text-align: center;
   vertical-align: middle;
   min-width: 100px;
```

Comment:
I don't think any of @dturner32 's mockups have text short enough to demonstrate a min-width, so we should probably check with him that this is still desired.

Reply:
> I don't think any of @dturner32 's mockups have text short enough to demonstrate a min-width, so we should probably check with him that this is still desired.

With the 24px of l/r padding I'm going to assume that we can remove this since any text will create a large enough click target

Reply:
> > I don't think any of @dturner32 's mockups have text short enough to demonstrate a min-width, so we should probably check with him that this is still desired.
> 
> With the 24px of l/r padding I'm going to assume that we can remove this since any text will create a large enough click target

https://github.com/sonatype/sonatype-react-shared-components/pull/39/commits/fff7ae93ed8ea9dfefb1e1b544413219a37e8032